src/sentry/rules/processing/processor.py

```diff
@@ -290,7 +333,8 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
             predicate_iter = (self.condition_matches(f, state, rule) for f in predicate_list)
             predicate_func = get_match_function(match)
             if predicate_func:
-                if not predicate_func(predicate_iter):
+                if not predicate_func(predicate_iter) and not slow_conditions:
```

Comment:
`and  process_slow_conditions_later and not slow_conditions`? I guess that's kind of handled later too